.circleci/config.yml

```diff
@@ -69,7 +46,6 @@ jobs:
           password: $DOCKERHUB_PASSWORD
     steps:
       - setup_nightly_tests
-      - install_cypress_dependencies
```

Comment:
`setup_nightly_tests` calls `setup_integration_test` which calls `install_cypress_dependencies`, so removed that here and below